src/pages/AddLiquidity/APRCalculation.tsx

```diff
@@ -33,12 +33,11 @@ export const apr = (
   const tokenAPrice = tokenBreserve.div(tokenAreserve)
   // we consider the values of tokenBReserve and tokenAreserve are the same
   const totalLp = tokenBreserve.times(2)
-  const rewardPerWeek = new BN(rewardPerBlock.toString()).times(blocksPerWeek)
-
-  const rewardPerWeekMeasuredByTokenB = rewardPerWeek.times(tokenAPrice)
+  const rewardPerWeek = new BN(rewardPerBlock.toString()).times(54000)
```

Comment:
Should it be 7200 * 7 = 50400.
Other than this, I am ok to merge

Reply:
Yes, you are right. I miss-typed it. Fixed.